contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -2474,7 +2476,8 @@ BEGIN
         END IF;
     END IF;
 
-    IF fully_supported = 'f' AND CURRENT_USER IN('dbo', 'master_dbo', 'tempdb_dbo', 'msdb_dbo') THEN
+    IF fully_supported = 'f' AND
+		(SELECT orig_username FROM sys.babelfish_authid_user_ext WHERE rolname = CURRENT_USER) = 'dbo' THEN
```

Comment:
With this change, the current user can be from the user created databases too i.e., "db1_dbo". Is that expected? If yes, then it's fine.

Reply:
Yes.